centaur/src/main/resources/standardTestCases/sub_workflow_decls/sub_workflow_decls.wdl

```diff
@@ -11,15 +11,13 @@ workflow sub_workflow_decls {
   String workflow_overridden_input = select_first([workflow_overridden_input_optional, workflow_default_input_optional])
 
   call decls.sub_decls as sudecls { input:
-    needs_to_be_supplied = "initialized"
-    , default_which_is_overridden = "overridden"
-    , overridden_depends_on_needs_to_be_supplied = "you've been overridden!"
-    , overridden_depends_on_default_which_is_overridden = "your override has been overridden"
-    #, optional_without_default_supplied = "consider yourself supplied"
-    , optional_with_default_but_overridden = "supplied but overridden optional"
-    , passthrough_required_input = workflow_required_input
-    , passthrough_initialized_input = workflow_initialized_input
-    , passthrough_overridden_input = workflow_overridden_input
+    needs_to_be_supplied = "initialized",
+    default_which_is_overridden = "overridden",
+    optional_without_default_supplied = "consider yourself supplied",
+    optional_with_default_but_overridden = "supplied but overridden optional",
+    passthrough_required_input = workflow_required_input,
```

Comment:
goofed up highlighting

Reply:
The really sad thing is that the ‘input’ keyword at the top has not been treated to the en-reddening in the same way 😧